data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -1673,6 +1673,7 @@
     "name": { "str_sp": "parasaurolophus" },
     "copy-from": "mon_hadrosaurus",
     "//": "When large weights and volumes are fixed this should be 4000 L and 4000 kg",
+    "bodytype": "bear",
```

Comment:
Huh, this should be automagically handled by the copy-from line. It was an elephant in testing for you?